### DISCUSSION TITLE
How to configure additional IP addresses

### DISCUSSION THREAD

**Discussion by gigbuilder** [postalserver/postal#2356] [Help with using Postal]
I've looked, I've searched, I cannot find an answer.  Using Postal there are two servers - one for each of my companies.  The original IP for the machine works fantastic -- x.x.x.38 , but when I add a second IP (using ip pools) and assign x.x.x.39 the email tries to send from the proper address but, of course, fails due to ptr issues.  Postal server, regardless of what "server" you add and regardless of IP address gives me the same information when checking the domain -- and it shows correct.  I have a reverse PTR entry for .38 and .39 yet when I try to send mail I receive the following (at least from Gmail).  I assume the credential check for the second domain is bugging out.  As I mentioned, there IS a PTR record for each ip address.

550-5.7.25 [98.143.16.39] The IP address sending this message does not have a 550-5.7.25 PTR record setup, or the corresponding forward DNS entry does not 550-5.7.25 point to the sending IP. As a policy, Gmail does not accept messages 550-5.7.25 from IPs with missing PTR records. Please visit 550-5.7.25 https://support.google.com/mail/answer/81126#ip-practices for more 550 5.7.25 information. e21-20020a05683013d500b0068bdb2e6936si7720608otq.210 - gsmtp

**Comment by willpower232** [comment]
Is it possible you're also accidentally using IPv6 as well so there is another PTR record missing?

Otherwise, it would be good to use a reverse lookup tool like mxtoolbox.com to verify that both your PTR records definitely exist and are visible by the rest of the internet.
